packages/ai-jsx/src/core/conversation.tsx

```diff
@@ -204,6 +204,8 @@ function toConversationMessages(partialRendering: AI.PartiallyRendered[]): Conve
 export async function renderToConversation(
   conversation: AI.Node,
   render: AI.ComponentContext['render'],
+  logger?: AI.ComponentContext['logger'],
+  logType?: 'prompt' | 'completion',
```

Comment:
What's the semantic difference between `prompt` and `completion`?

Reply:
Ah, it's the input and output?

Reply:
Right, specifically it indicates that the conversation is the input to/output from a chat model.